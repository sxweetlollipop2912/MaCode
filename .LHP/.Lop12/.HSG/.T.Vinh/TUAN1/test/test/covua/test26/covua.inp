30
xd01011.1d111dddd0d.0.d01d..10
dx.0dd11101d.0d.d000dd1d.1dd0d
1.x01dd0101d10.1d0000dd.11dd.1
011xd01010110.0d0d111d01d1d1.0
1d0dxddd0.1d..1.0.1..1d1100100
0dd1dxd.0.0.0d0d10d1000111d101
00d0ddx10d.d01dd10d0.d10d111dd
.011d.0x1d.0dd.ddd010000011110
00001110xd11010d00d1d1dd0.01d0
d111..dddxd1dd00dddd.d000d10dd
000001..0dxd100.d0.10111.1d..1
0dd0d.d100dxd11110d100dd1000d1
0.01.11d1d0dxd010.00d0d11000.0
d11..d0d0d10dx.01000d.d1000.d0
dd.101d.11101.x.ddd1.d1.1d1dd1
d.0d.dddd1.001.x.1.00.01d1d1d.
ddd1100d1dd010d.x1..0d01101.00
111d.11d1d11.1d00x00d10d.00.01
d1100dd1dd.d11d..1x00d..1d0d0d
.110.0100d001101.11xd00.1d0d10
1d10.1.1d.11dd.11d1dxd0dd01.11
.ddd01d10d011.d.d0d1dx.d.1dd11
d0d1d101d10ddd0111.11.xd1.dd0.
1d.00011d10d00.00d..dddxdd0d11
0.0d00d111.0010d0.00d.0dxdd10d
d0001000.d0111d011dd10.ddx1010
.ddd1d0010d1110d01110dd1d0x010
.dd0000001.11.d0..dd.ddd011x.0
01..11d0dd.d.ddd11100010100.x0
1d0110d11d00110.10d100.0d1111x