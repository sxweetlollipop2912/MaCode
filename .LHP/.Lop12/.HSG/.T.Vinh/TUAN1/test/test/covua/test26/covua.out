1 3 4 5 11 15 16 18 19 21 22 23 30
